doc/tutorial/first_steps.md

```diff
@@ -0,0 +1,290 @@
+(first-steps)=
+# First steps with LXD
+
+This tutorial guides you through the first steps with LXD.
+It covers installing and initializing LXD, creating and configuring some instances, interacting with the instances, and creating snapshots.
+
+After going through these steps, you will have a general idea of how to use LXD, and you're ready to explore more advanced use cases!
```

Comment:
There seems to be a tense change here ("you will have" then ""you're ready"). To the future tense sounds better IMHO.

Reply:
Not a fan of using future tense too much in the docs, but I agree this sounded a bit weird. ;)